libs/ui-watt/src/lib/components/button/watt-button.component.html

```diff
@@ -23,6 +23,26 @@
 ></ng-container>
 
 <div #contentWrapper class="content-grid">
+  <ng-container *ngTemplateOutlet="contentWrapperTemplate"></ng-container>
+</div>
+
+<ng-template #contentWrapperTemplate>
+  <ng-container *ngIf="loading; else buttonContentTemplate">
```

Comment:
When both templates is using "buttonContentTemplate" considering just having a ngIf on the spinner, it would be easier to follow.

Reply:
The two templates are not completely identical. The `loadingTemplate` wraps `buttonContentTemplate` in an extra `div.content-grid.content-grid-item` element. That is why `buttonContentTemplate` is moved on its own.